src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -22,10 +22,23 @@
 #include <ripple/app/main/Application.h>
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/data/DatabaseCon.h>
+#include <ripple/core/ConfigSections.h>
 #include <boost/format.hpp>
+#include <boost/tokenizer.hpp>
 
 namespace ripple {
 
+namespace detail
+{
+/** Amendments that are "compiled into" this app
+
+    Add amendments to this collection at build time to enable them on this
+   server. The first element of the pair is the id, the second element is
+   the "friendly name".
+*/
+std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
```

Comment:
@rec Changing from a pair<string,string> to a struct (instead of a pair<uint256,string>) addresses this. I can check for a valid hex string in the ::ctor of this class. Will change.